admin/Default/manage_post_editors.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+$template->assign('PageTopic', 'Manage Galactic Post Editors');
+
+// Get the list of active games ordered by reverse start date
+$activeGames = array();
+$db_time = $db->escapeNumber(TIME);
+$db->query('SELECT game_id, game_name FROM game WHERE start_date < ' . $db_time . ' AND end_date > ' . $db_time . ' ORDER BY start_date DESC');
+while ($db->nextRecord()) {
+	$activeGames[] = array('game_name' => $db->getField('game_name'),
+	                       'game_id' => $db->getInt('game_id'));
+}
+$template->assign('ActiveGames', $activeGames);
+
+if ($activeGames) {
+	// Set the selected game (or the first in the list if not selected yet)
+	if (isset($_POST['game_id'])) {
+		SmrSession::updateVar('selected_game_id', $_POST['game_id']);
```

Comment:
The list of possible `game_id`'s that they can select from in the dropdown menu is populated by a database query (`$activeGames`), so in that sense `$_POST['game_id']` must provide a valid game ID. If they manipulate the form to be able to submit a game ID that was not provided... well then perhaps they deserve to get a database error. ;)